azext_iot/central/_help.py

```diff
@@ -1752,3 +1754,282 @@ def _load_central_edge_help():
             --device-id {deviceId}
             --children-ids {child_1} {child_2}
     """
+
+
+def _load_central_enrollment_group_help():
+    helps[
+        "iot central enrollment-group"
+    ] = """
+          type: group
+          short-summary: Manage and configure IoT Central enrollment group
+      """
+
+    helps[
+        "iot central enrollment-group list"
+    ] = """
+    type: command
+    short-summary: Get the list of enrollment groups in an application
+    examples:
+      - name: Get the list of enrollment groups in an application
+        text: >
+          az iot central enrollment-group list
+          --app-id {appid}
+    """
+
+    helps[
+        "iot central enrollment-group show"
+    ] = """
+    type: command
+    short-summary: Get details about an enrollment group by ID
+    examples:
+      - name: Get details about an enrollment group by ID
+        text: >
+          az iot central enrollment-group show
+          --app-id {appid}
+          --id {enrollmentGroupId}
+    """
+
+    helps[
+        "iot central enrollment-group delete"
+    ] = """
+    type: command
+    short-summary: Delete an enrollment group by ID
+    examples:
+      - name: Delete an enrollment group by ID
+        text: >
+          az iot central enrollment-group delete
+          --app-id {appid}
+          --id {enrollmentGroupId}
+    """
+
+    helps[
+        "iot central enrollment-group create"
+    ] = """
+    type: command
+    short-summary: Create an enrollment group
+    examples:
+      - name: Create an enrollment group
+        text: >
+          az iot central enrollment-group create
+          --app-id {appid}
+          --id {enrollmentGroupId}
+          --at {attestation}
+          --display-name {displayName}
+          --type {type}
+    """
+
+    helps[
+        "iot central enrollment-group update"
+    ] = """
+    type: command
+    short-summary: Update an enrollment group
+    examples:
+      - name: Update an enrollment group
+        text: >
+          az iot central enrollment-group update
+          --app-id {appid}
+          --id {enrollmentGroupId}
+          --at {attestation}
+          --display-name {displayName}
+          --type {type}
+          --enabled true
+    """
+
+    helps[
+        "iot central enrollment-group x509"
+    ] = """
+          type: group
+          short-summary: Manage and configure IoT Central enrollment group x509 certificate
```

Comment:
what is the difference between the enrollment groups from
`iot central enrollment-group`
and 
`iot central enrollment-group x509`
As in, would a customer create an enrollment with the first command and then set the certificate with the second? DPS enrollment groups can be created with x509 certificates or sym keys from one command

Reply:
Customer will create an enrollment with either x509 or sym key using first command and second command will set up primary or secondary x509 certificate. If we put these into one command, we need to call 2 central APIs for one command which might be weird. @yang-hai-feng Any thoughts?

Reply:
Hi Hangyi, I am not concerning the multiple API call in one command, let's make it easier for customer to use the command, agreed with @vilit1, we better to algin the pattern in CLI from central and dps.  [Check Dps docs](https://docs.microsoft.com/en-us/cli/azure/iot/dps/enrollment-group?view=azure-cli-latest#az-iot-dps-enrollment-group-create)

Reply:
Sure. Then I'll combine them.

Reply:
Merge them into one command.